docs/stdlib/objects.rst

```diff
@@ -43,19 +41,17 @@ which is a subtype of ``std::BaseObject``.
             required readonly link __type__ -> schema::ObjectType;
         }
 
-    Subtypes may override the ``id`` property, but only to valid uuid
-    generation functions. Currently, these
-    are :eql:func:`uuid_generate_v1mc` and :eql:func:`uuid_generate_v4`.
+    Some subtypes may override the ``id`` property, but only to validate
+    UUID generation functions. Currently, these include
+    :eql:func:`uuid_generate_v1mc` and :eql:func:`uuid_generate_v4`.
```

Comment:
```suggestion
    Subtypes may override the ``id`` property, but only with a valid UUID
    generation function. Currently, these are
    :eql:func:`uuid_generate_v1mc` and :eql:func:`uuid_generate_v4`.
```

I recommend mostly reverting here. The rewrite introduces some issues.
- "Some" is already implied by "may" (not all of them do it), so it doesn't add anything here.
- The ``id`` is not overridden for the purpose of validating a UUID generation function but is overridden _to_ a valid UUID generation function. "with" might actually be clearer here than "to," so I'll incorporate that into my suggestion.
- Swapping "are" for "include" implies there may be other valid UUID functions the user can override their ID with, which I don't believe is the case.